python/lsst/ts/MTMount/constants.py

```diff
@@ -28,7 +29,29 @@
 # Update this when Tekniker adds a dedicated pair of ports for the CSC.
 CSC_COMMAND_PORT = HHD_COMMAND_PORT
 
-TELEMETRY_PORT = 50036
+TELEMETRY_PORT = 50035
```

Comment:
I am a bit anxious about having these ports as "constants", as they can easily be changed in configuration.

Reply:
I would rather live with this for now. They should be hard coded and left alone. The change shown was just because Tekniker made a mistake in the CCW. Basically I don't see any use case for changing the ports in the low-level controller, so I don't see any point to making them a configuration parameter here.